inc/Calculator.h

```diff
@@ -11,6 +11,8 @@
 #define CALCULATOR_H
 
 #include <iostream>
-// Add your function declarations here.
+// Add your function declarations here test.
+    void runCalc();
 
+    void calc(char calcType, double num1, double num2);
```

Comment:
1- similar to missing doxygen comments
2- if the arguments are not change, can be passed by const & for constant reference